cli/dcoscli/cluster/main.py

```diff
@@ -123,14 +123,19 @@ def _list(json_, attached):
     return
 
 
-def _remove(name):
+def _remove(name, all=False):
     """
     :param name: name of cluster
     :type name: str
+    :param all: remove all clusters if True
+    :type all: bool
     :rtype: None
     """
-
-    return cluster.remove(name)
+    if all:
+        for c in cluster.get_clusters():
+            cluster.remove(c.get_name())
+    else:
+        return cluster.remove(name)
```

Comment:
No need to return this, unless `cluster.remove` has a return type and then the `:rtype:` documentation here needs changing.